### ISSUE TITLE
Upgrade to Bootstrap 5

### ISSUE THREAD

**Issue by thatbudakguy** [geoblacklight/geoblacklight#1373]
### Descriptive summary

This is a [framework recommendation](https://geoblacklight.org/docs/framework-recommendations/) for GBL v5, and also a requirement for upgrading to Blacklight 8.

### Related work

Work to move a lot of the views to View Components (see #1270) probably needs to be done first, otherwise there will be merge conflicts when the class names are changed in templates.

**Comment by jcoyne** [comment]
Bootstrap 4 is EOL as of Jan 1, 2023 https://getbootstrap.com/docs/4.6/end-of-life/ so this is definitely a priority.

**Comment by ewlarson** [comment]
Reopening this issue as all the Devise views are not Bootstrap'd. Seems also not Bootstrap'd upstream in BL — Is this intended?

### Log in
![Screenshot 2024-08-08 at 2 07 15 PM](https://github.com/user-attachments/assets/a81a52d5-e1c9-4fd8-ac6d-6d982e5860b7)

### Sign up
![Screenshot 2024-08-08 at 2 07 28 PM](https://github.com/user-attachments/assets/ae61489a-1f35-4f1a-9e76-de5930544ecb)

### Edit User
![Screenshot 2024-08-08 at 2 07 07 PM](https://github.com/user-attachments/assets/4c111706-d829-42a4-9da8-f3490dee2664)

### Forgot password
![Screenshot 2024-08-08 at 2 07 20 PM](https://github.com/user-attachments/assets/636893f2-6e2f-406b-ac95-070d61899cd2)

**Comment by thatbudakguy** [comment]
Should this become an issue in BL?

**Comment by thatbudakguy** [comment]
Closing in favor of #1509
